src/contents/button.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import classes from "./styles/button.module.scss";
+
+const button = (props) => (
```

Comment:
Component names should always be `Capital` like `Button`. Also, give it a unique name as Button is generic.
Something like `AccodeButton`, ya aisa kuch, anything.